bot.py

```diff
@@ -39,6 +39,30 @@ async def popcorn(interaction: nextcord.Interaction):
         response += f"but {kernels} of them are just kernels!"
     await interaction.response.send_message(response)
 
+# Help Command
+
+@bot.slash_command(guild_ids=[GUILD_ID], name="help", description="lists all commands and descriptions")
+async def help(interaction: nextcord.Interaction):
+    response = ""
```

Comment:
Please reformat message response so that after each command and each description there is a new line. Something like:

/watchlist_create
- create a new watchlist

/watchlist_delete
- delete an existing watchlist

this would require a new line after the command and then 2 new lines after the description.

Reply:
ignore - make change on help command pull request.